### PR TITLE
Generate key

### PR DESCRIPTION
**Pull request by surabhishukla**

### CODE REVIEW

File: src/Pages/License/Generate/GenerateProductKeyDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import { CustomCard, CustomCardContent } from "./GenerateLicense";
+import CheckIcon from "@mui/icons-material/Check";
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
+import { useState } from "react";
+
+const GeneratrProductKeyDetails = () => {
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/GenerateProductKeyDetails.jsx:8]: Fix typo

File: src/Pages/License/Generate/GenerateLicense.jsx [hunk 2]
```diff
@@ -61,6 +61,9 @@ const GenerateLicense = (props) => {
         obj.btn.action = () => navigate("/dashboard");
         obj.page = null;
         break;
+      case 4:
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/GenerateLicense.jsx:64]: Fix case no.

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -22,4 +22,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 package-lock.json
-package-lock.json
```
[the remark this answers is not in the document]
Comment (surabhishukla) [reply .gitignore:25]: done

File: src/Pages/License/Generate/GenerateLicense.jsx (hunk 2 above)
Comment (surabhishukla) [reply src/Pages/License/Generate/GenerateLicense.jsx]: done

File: src/Pages/License/Generate/GenerateProductKeyDetails.jsx (hunk 1 above)
Comment (surabhishukla) [reply src/Pages/License/Generate/GenerateProductKeyDetails.jsx]: done

**Review by pinak-globallogic — APPROVED** [review]

**Pull request merged**